ui-packages/packages/runtime-tools-dev-ui-webapp/server/server.js

```diff
@@ -95,7 +95,8 @@ app.get(
 app.get('/:processId/:taskName/schema', controller.getTaskDefinitionForm);
 
 app.get('/forms/list', controller.getForms);
-
+app.get('/customDashboard/list', controller.getCustomDashboards);
```

Comment:
```suggestion
app.get('/customDashboards/list', controller.getCustomDashboards);
```

Please change it in all places that use this name in a path to keep the coherence with other resources, like forms and processes.

Reply:
I think list means multiple dashboard, so I remove 's'.